source/memory/memory.go

```diff
@@ -49,5 +69,15 @@ func (s Source) Discover() (source.Features, error) {
 		features["numa"] = true
 	}
 
+	// Check kernel memory compaction
+	choice, err := GetChoice("/sys/kernel/mm/transparent_hugepage/enabled")
+	if err != nil {
+		return nil, fmt.Errorf("can't read /sys/kernel/mm/transparent_hugepage/enabled: %s", err.Error())
+	}
+
+	// File content is supposed to be:
+	// [always] madvise never
+	features["transparent_hugepage"] = choice
```

Comment:
I would use transparent_hugepage.enabled here. There might be need to be other attributes in the future.

Reply:
Done.